docs/guides/manage-identities/01_overview.mdx

```diff
@@ -0,0 +1,65 @@
+---
+id: overview
+title: What is an Identity?
+sidebar_label: Overview
+---
+
+Ory Identity Service (Kratos) uses Identities to identify entities that sign up or authenticate. These can be humans that sign up
+on a website or an application that uses Ory, for example online store customers, file sharing service users, or company
+contractors signing up to use internal systems.
+
+The Identity isn't limited to humans. Ory can create Identities for programmatic entities such as IoT devices, client
```

Comment:
```suggestion
The identity isn't limited to humans. Ory can create identities for programmatic entities such as IoT devices, client
```

IMO it looks more natural to have it lowercase. I get that you want to use "Identity" as a proper term in the Ory context, but IMO its still clear if lowercase and looks less weird.
(at least when its not "Ory Identities")

Reply:
Thanks! I think you're right. I decided to do `identity` and `Identity Schema`.